package com.javaExceptionHandlingWithtryCatchFinally;

public class Example1 {

	public static int calculate(int x, int y)
	{
		return x/y;
	}

}
